backend/fms_core/services/library.py

```diff
@@ -143,3 +143,23 @@ def convert_library(process, platform, sample_source, container_destination, coo
             errors.append(e)
 
     return library_destination, errors, warnings
+    
+def set_library_size(sample, library_size = None):
```

Comment:
I would create a more multipurpose function here ... update_library(sample, library_data) maybe ... Library_data would hold a dictionary of values to update in the model. That way you can omit attributes if you don't want to change them or pass None if you want to set it to None. We are likely to have to set many values in the library over the time and it would be easier to have a more generic call instead. Maybe kwargs could be used too might be more elegant and easy to use.